src/pages/staticPages/tools/oncoprinter/OncoprinterGeneticUtils.ts

```diff
@@ -42,6 +42,9 @@ import {
     MUTATION_STATUS_GERMLINE,
     PUTATIVE_DRIVER,
 } from '../../../../shared/constants';
+import { AnnotateCopyNumberAlterationQuery } from 'cbioportal-frontend-commons/src/api/generated/OncoKbAPI';
+import { EvidenceType, IOncoKbData } from 'cbioportal-frontend-commons';
+import { generateCopyNumberAlterationQuery } from 'cbioportal-frontend-commons/src/lib/oncokb/OncoKbUtils';
```

Comment:
group all of these imports into a single `import { ... } from 'cbioportal-frontend-commons'`?